solr/solr-ref-guide/build.gradle

```diff
@@ -44,13 +44,13 @@ buildscript {
     }
 
     dependencies {
-        classpath "org.asciidoctor:asciidoctorj:1.6.2"
+        classpath "org.asciidoctor:asciidoctorj:2.4.1"
     }
 }
 
 plugins {
     id 'java'
-    id 'com.github.jruby-gradle.base' version '2.0.0-alpha.7'
```

Comment:
This will conflict with the cleanup Chris is doing - I modified the same thing there.